src/main/webapp/instructorHelp.jsp

```diff
@@ -1914,62 +1914,147 @@
                                 <div class="col-sm-12 padding-15px margin-bottom-15px background-color-light-blue">
                                     <div>
-                                        <textarea rows="5" class="form-control textvalue nonDestructive" name="questiontext" id="questiontext-1" data-toggle="tooltip" data-placement="top" tabindex="9" title="Please enter the question for users to give feedback about. e.g. What is the biggest weakness of the presented product?">Tutor's comments about the team presentation</textarea>
+                                        <textarea rows="5" class="form-control textvalue nonDestructive" name="questiontext" id="questiontext-1" data-toggle="tooltip" data-placement="top" tabindex="9" title="Please enter the question for users to give feedback about. e.g. What is the biggest weakness of the presented product?">Tutor's comments about the class in general</textarea>
                                     </div>
 
                                 </div>
                                 <br>
                                 <div class="col-sm-12 padding-15px margin-bottom-15px background-color-light-green">
-                                    <div class="col-sm-12 padding-0">
-                                        <b>Feedback Path</b> (Who is giving feedback to whom?)
+                                    <div class="col-sm-12 padding-0 margin-bottom-7px">
+                                        <b>Feedback Path</b> (Who is giving feedback about whom?)
                                     </div>
-                                    <div class="col-sm-6 padding-0" data-toggle="tooltip" data-placement="top" title="Who will give feedback">
-                                        <label class="col-sm-5 control-label">
-                                            Who will give the feedback:
-                                        </label>
-                                        <div class="col-sm-7">
-                                            <select class="form-control participantSelect" name="givertype" id="givertype-1">
-                                                <option value="SELF">Me (Session creator)</option>
-                                                <option value="STUDENTS">Students in this course</option>
-                                                <option value="INSTRUCTORS" selected="">Instructors in this course</option>
-                                                <option value="TEAMS">Teams in this course</option>
-
-                                            </select>
-                                        </div>
+                                    <div class="feedback-path-dropdown btn-group">
+                                        <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                            Predefined combinations:
```

Comment:
This is the very reason why I call `instructorHelp.jsp` a "dreadful page" -- we have to hardcode everything that can be rendered with tags. But until we go to that direction, we don't really have another choice here.